modules/ipi-install-node-requirements.adoc

```diff
@@ -10,7 +10,7 @@ IPI installation involves a number of hardware node requirements:
 
 - **CPU architecture:** All nodes must use `x86_64` CPU architecture.
 
-- **Similar nodes:** Nodes must have an identical configuration per role. That is, control plane nodes must be the same brand and model with the same CPU, RAM and storage configuration. Worker nodes must be identical.
+- **Similar nodes:** We recommend nodes have an identical configuration per role. That is, we recommend nodes be the same brand and model with the same CPU, RAM and storage configuration.
```

Comment:
s/we recommend/Red Hat recommends
CPU, Ram, and storage

Reply:
It is recommended?

Reply:
Ram shouldn't be RAM as Random Access Memory (it's an acronym)

Reply:
~~~
- **Similar nodes:** It is recommended for nodes to have an identical configuration per role. That is, being the same brand and model with the same CPU, RAM and storage configuration.
~~~

Our upstream wording

Reply:
The upstream wording uses passive voice.

Reply:
From my KCS writing times, I recall the guide was to use passive and avoid "you must do XXX, then you must do YYY" and use passive.

I'm free to keep whatever you prefer, but my concern here is that if we're talking upstream and we use 'vars' for things like product name, we should do the same with 'Red Hat' and use some var for that (and decide what to put in upstream)